migrate-to-hugo.md

```diff
@@ -0,0 +1,88 @@
+## Website Elements
+
+- Title
+- Description
+- Tabs that switches between categories
+  - Apps
+  - Tools (difference from apps?)
+  - Archives
+  - Videos
+  - Articles
+  - Discussions
+- Below, after clicking category, show most popular tag within that category
+  and allow further filtering. (future allow multiple tags? Will be hard without JS)
+  - topic:blockchain
+  - topic:ethereum
+  - topic:research
+  - topic:file-sharing
+  - topic:chat
+  - state:prototype
+  - state:beta
+  - state:stable
+  - platform:firefox
+  - platform:chrome
+  - platform:windows
+- List of items matching the selected filters ("apps" + "topic:chat" shows all chat apps)
+- Each item gets a card
+  - layout can be X amount of cards depending on screen width
+  - each card has:
+    - title
+    - screenshot of item in action
+    - first 80 characters of description
+    - highlighted and on the top if `featured`
+    - the title+description are normal anchor tags
+    - Edit link for linking to the data file (and possibly row) in Github
+      for easy editing
+- Link to repository
+- Link to issue/markdown file describing how to suggest new items
+- Link to `Ecosystem` category on discuss.ipfs.io
+
+## Data Structure
+
+- Each category has it's own data file within `data/`
+- Title for each item has to be unique (category should not be in URL)
+- Screenshot location is based on title. If no screenshot found, should use a default image
+
+Example `apps.yml`
+
+```
+title: 'Apps'
+description: 'Applications built with IPFS or with functionality that uses IPFS'
+items:
+  - title: Akasha
+    description: A Next-Generation Social Media Network, powered by Ethereum and embedded into IPFS
+    featured: true
```

Comment:
I would replace an explicit `featured` boolean flag with a list of tags, and `featured` being just one of tags.
Based on tags, we can do  all sorts of styling/filtering in the UI.

Reply:
Yeah, that's a solid idea.